Tests/IntegrationTests/SearchIntegrationTests.swift

```diff
@@ -17,74 +17,79 @@ import XCTest
 
 
 class SearchIntegrationTests: XCTestCase {
-
+    
     var searchTester: SearchTester?
-
+    
     override func tearDown() {
         super.tearDown()
         searchTester = nil
     }
 
     func testSearchingGoldenBoys() {
-        let findExpectation = expectation(description: "find expectation")
+        weak var findExpectation = expectation(description: "find expectation")
```

Comment:
I got the following error before: `API violation - multiple calls made to -[XCTestExpectation fulfill] for find expectation.`